infra/terraform/global/main.tf

```diff
@@ -65,3 +65,41 @@ module "hmpps_oasys_upload_user" {
   org_name          = "hmpps"
   system_name       = "oasys"
 }
+
+// Backup etcd volumes attached to kubernetes masters -->
+
+variable "environment_variables" {
+  type = "map"
+
+  default = {
+    "TAG_KEY"            = "etcd"
+    "TAG_VALUE"          = "1"
+    "INSTANCE_TAG_KEY"   = "k8s.io/role/master"
+    "INSTANCE_TAG_VALUE" = "1"
+  }
+}
+
+# Create Snapshot policy document
+data "template_file" "lambda_create_snapshot_policy" {
+  template = "${file("assets/create_etcd_ebs_snapshot/lambda_create_snapshot_policy.json")}"
+}
+
+# Lambda requires that we zip the distribution in order to deploy it
+data "archive_file" "kubernetes_etcd_ebs_snapshot_code" {
+  source_file = "assets/create_etcd_ebs_snapshot/create_etcd_ebs_snapshot"
+  output_path = "assets/create_etcd_ebs_snapshot/create_etcd_ebs_snapshot.zip"
+  type        = "zip"
+}
+
+module "kubernetes_etcd_ebs_snapshot" {
+  source                = "../modules/lambda_mgmt"
+  lambda_function_name  = "create_etcd_ebs_snapshot"
+  zipfile               = "assets/create_etcd_ebs_snapshot/create_etcd_ebs_snapshot.zip"
+  handler               = "create_etcd_ebs_snapshot"
+  source_code_hash      = "${data.archive_file.kubernetes_etcd_ebs_snapshot_code.output_base64sha256}"
+  lamda_policy          = "${data.template_file.lambda_create_snapshot_policy.rendered}"
+  environment_variables = "${var.environment_variables}"
+}
+
+// -->
```

Comment:
What is this comment for?

Reply:
Boundary to show where related resources end.  I thought it'd helpful since seemingly unrelated resoueces share the same file.